packs/remedy/sensors/workorder_sensor.py

```diff
@@ -0,0 +1,71 @@
+from st2reactor.sensor import PollingSensor
+from actions.lib.actions import BaseAction
+from actions import *
+
+class SimplePollingSensor(PollingSensor):
+    """
+    * self._sensor_service
+        - provides utilities like
+            get_logger() for writing to logs.
+            dispatch() for dispatching triggers into the system.
+    * self._config
+        - contains configuration that was specified as
+          config.yaml in the pack.
+    * self._poll_interval
+        - indicates the interval between two successive poll() calls.
+    """
+
+    def setup(self):
+        # Setup stuff goes here. For example, you might establish connections
+        # to external system once and reuse it. This is called only once by the system.
+        pass
+
+    def poll(self):
+        # This is where the crux of the sensor work goes.
+        # This is called every self._poll_interval.
+        # For example, let's assume you want to query ec2 and get
+        # health information about your instances:
+        #   some_data = aws_client.get('')
+        #   payload = self._to_payload(some_data)
+        #   # _to_triggers is something you'd write to convert the data format you have
+        #   # into a standard python dictionary. This should follow the payload schema
+        #   # registered for the trigger.
+        #   self._sensor_service.dispatch(trigger, payload)
+        #   # You can refer to the trigger as dict
+        #   # { "name": ${trigger_name}, "pack": ${trigger_pack} }
+        #   # or just simply by reference as string.
+        #   # i.e. dispatch(${trigger_pack}.${trigger_name}, payload)
+        #   # E.g.: dispatch('examples.foo_sensor', {'k1': 'stuff', 'k2': 'foo'})
+        #   # trace_tag is a tag you would like to associate with the dispacthed TriggerInstance
+        #   # Typically the trace_tag is unique and a reference to an external event.
+        entries = self.client.query(schema='TESTE_RENAN', qualifier='1=1', 
+                                    fields=['Request ID', 'Short Description', 'Status'], 0, 5)
+        
+        for entry_id, entry_values in entries:
+            
+            #self.client.update()
+            print('Entry ID: {}'.format(entry_id))
+            print('-' * 80)
+            for field, value in entry_values.items():
+                print('{}: {}'.format(field, value))
+        print()
+

```

Comment:
At the end of this method you likely want to dispatch a few trigger. Seems like this sensor is still WIP.